### ISSUE TITLE
OpenSSL CA certs not generated in latest tags

### ISSUE THREAD

**Issue by beltran-rubo** [vmware/photon#1032]
Hi,

The CA certs are not properly generated in the latest tags of the Container images. It was working properly in June and previous versions. I was able to reproduce the issue with the following tags:

```
docker pull -it photon bash
ls /etc/ssl/certs/*.0
ls: /etc/ssl/certs/*.0: No such file or directory
```

```
docker run -it photon:3.0-20200609 bash
ls /etc/ssl/certs/*.0
/etc/ssl/certs/02265526.0  /etc/ssl/certs/4d4ba017.0  /etc/ssl/certs/b204d74a.0
/etc/ssl/certs/03179a64.0 ...
```

In order to fix that it is required to generate those links:
```
tdnf install -y openssl-c_rehash
rehash_ca_certificates.sh
```

Do you plan to address this in the new versions of the base image?

**Comment by beltran-rubo** [comment]
This breaks runtimes that requires the CA certs from the system E.g.

```
ruby -rnet/https -e "Net::HTTP.get URI('https://github.com')"

Traceback (most recent call last):
    8: from -e:1:in `<main>'
    7: from /ruby/lib/ruby/2.5.0/net/http.rb:462:in `get'
    6: from /ruby/lib/ruby/2.5.0/net/http.rb:485:in `get_response'
    5: from /ruby/lib/ruby/2.5.0/net/http.rb:609:in `start'
    4: from /ruby/lib/ruby/2.5.0/net/http.rb:909:in `start'
    3: from /ruby/lib/ruby/2.5.0/net/http.rb:920:in `do_start'
    2: from /ruby/lib/ruby/2.5.0/net/http.rb:985:in `connect'
    1: from /ruby/lib/ruby/2.5.0/net/protocol.rb:44:in `ssl_socket_connect'
/ruby/lib/ruby/2.5.0/net/protocol.rb:44:in `connect_nonblock': SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate) (OpenSSL::SSL::SSLError)
```

**Comment by suezzelur** [comment]
@beltran-rubo thanks for reporting this, we will fix the base image and re-upload

**Comment by michellew-vmware** [comment]
Thank you for reporting this issue to us. 

We just fixed it and send a PR to docker team.
PR: https://github.com/docker-library/official-images/pull/8404/files

Once the above PR is merged, you can try to pull photon:3.0 and it should work.

Thanks,
Michelle

**Comment by beltran-rubo** [comment]
Thanks, we tested the new base image and it works properly.
